packages/framework/src/Model/Order/Messenger/HeurekaPlacedOrderMessageHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Order\Messenger;
+
+use Exception;
+use Psr\Log\LoggerInterface;
+use Shopsys\FrameworkBundle\Model\Order\OrderFacade;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+class HeurekaPlacedOrderMessageHandler implements MessageHandlerInterface
```

Comment:
Should this class be called Heureka? Other parts do not mention it and this could be universal for other actions like sending e-mail after order creation.

Reply:
I have two things that we should discuss:
- would not be better to have handlers configured manually as described in https://symfony.com/doc/5.4/messenger.html#manually-configuring-handlers? I know most of our colleagues do not like "magic" and this is one of those cases.
- if not, should not we follow the recommended way to define Handler in PHP 8 as described here https://symfony.com/doc/5.4/messenger.html#creating-a-message-handler using The #[AsMessageHandler] PHP attribute